Core/MIPS/MIPSTables.h

```diff
@@ -108,6 +93,28 @@ struct MIPSInfo {
 #define CDECL
 #endif
 
+struct MIPSInfo {
+	MIPSInfo() {
+		value = 0;
+		cycles = 0;
+	}
+
+	explicit MIPSInfo(u64 v, u16 c = 0) : value(v), cycles(c) {
+		if (c == 0) {
+			cycles = 1;
+			if (v & IS_VFPU)
+				cycles++;
+		}
+	}
+
+	u64 operator & (const u64 &arg) const {
+		return value & arg;
+	}
+
+	u64 value : 48;
+	u64 cycles : 64;
```

Comment:
what's the purpose of these bitfield specifiers? don't look necessary to me?

Reply:
Hm that's weird.  I was trying to just put it in the top 16 bits and could swear I put 16 here and even checked sizeof(MIPSInfo).

-[Unknown]